src/store/ducks/generic/jurisdictions.ts

```diff
@@ -58,6 +59,8 @@ export interface ZambiaIRSFocusArea extends GenericJurisdiction {
   spraytarg: number;
   spraysuccess: number;
   roomcov: number;
+  notsprayed_reasons: string[];
+  notsprayed_reasons_counts: Dictionary;
```

Comment:
I think that we can make this type more accurate.  Seems like it is Dictionary<number>